### PR TITLE
9 users can request space

### PR DESCRIPTION
**Pull request by SiAshbery**
Users Can Request Space
-As a user,
-So I have somewhere to stay,
-I want to request a space for one night.

-Includes helpers for sign_up and sign_in
-Adds buttons for each listed space
-Captures ID from selected space
-Saves ID to session
-Renders requested space name in home page

### CODE REVIEW

File: app/views/spaces/spaces_list.erb [hunk 1]
```diff
@@ -1,11 +1,16 @@
 <ul>
   <% @spaces.each do |space| %>
-    <li>
+    <li id='space ID: <%=space.id%>'>
```
Comment (charlottebrf) [inline app/views/spaces/spaces_list.erb]: Should this be:
```
 <li class="space" id='<%=space.id%>'>
```

File: app/views/spaces/spaces_list.erb [hunk 2]
```diff
@@ -1,11 +1,16 @@
...
       <h2><%=space.name%></h2>
       <br>
       <%=space.description%>
       <br>
       <%="£#{space.price} per night" %>
+      <br>
+      <form action='/bookings/new' method="POST">
```
Comment (charlottebrf) [inline app/views/spaces/spaces_list.erb]: HTML style conventions use "" instead of ''

File: app/views/users/home.erb [hunk 3]
```diff
@@ -15,3 +15,7 @@
       <input type="submit" value="Sign Up">
     </form>
 <% end %>
+
+<% if @requested_space %>
+  <p>You Have Requested <%= @requested_space.name%></p>
+Cretaed <% end %>
```
Comment (charlottebrf) [inline app/views/users/home.erb]: Is this meant to be here?  "Cretaed"

File: spec/web_helper.rb [hunk 4]
```diff
@@ -8,3 +8,23 @@ def new_space(name = "Amazingly Romantic Shed", description = "A shed that is ro
   fill_in "price", with: price
   click_button("submit")
 end
+
```
Comment (charlottebrf) [inline spec/web_helper.rb]: Nice!

File: app/views/users/home.erb (hunk 3 above)
Comment (SiAshbery) [inline app/views/users/home.erb]: This is a weird error. Ignore ME!

**Pull request closed without merging**